tus_storagehandler/app.py

```diff
@@ -0,0 +1,57 @@
+"""API server entry point."""
+
+import logging
+import os
+from pathlib import Path
+
+from connexion import FlaskApp  # type: ignore
+from dotenv import load_dotenv  # type: ignore
+from foca import Foca
+
+load_dotenv()
+logger = logging.getLogger(__name__)
+
+
+def init_app() -> FlaskApp:
+    """Initialize and return the FOCA app.
+
+    This function initializes the FOCA app by loading the configuration
+    from the environment variable `TUS_FOCA_CONFIG_PATH` if set, or from
+    the default path if not. It raises a `FileNotFoundError` if the
+    configuration file is not found.
+
+    Returns:
+        A Connexion application instance.
+
+    Raises:
+        FileNotFoundError: If the configuration file is not found.
+    """
+    # Determine the configuration path
+    config_path_env1 = os.getenv("TUS_FOCA_CONFIG_PATH")
+    print(config_path_env1)
+    if config_path_env := os.getenv("TUS_FOCA_CONFIG_PATH"):
+        print(config_path_env)
+        config_path = Path(config_path_env).resolve()
+    else:
+        config_path = (
+            Path(__file__).parents[1] / "deployment" / "config.yaml"
+        ).resolve()
```

Comment:
Tried running the server, this is all cool but you have put the config file in `tus_storagehandler`,  this might work when you have set the variable, project structure should follow a work with default setting. Ie please move `config.yaml` to `./deployement`.